src/NCDialog.cc

```diff
@@ -1081,10 +1081,17 @@ void NCDialog::processInput( int timeout_millisec )
 
 	    case WEOF:
 
+		if (pendingEvent)
+		{
+		    yuiDebug() << "Keeping the pending event" << std::endl;
```

Comment:
Similar to below: Sure you want to keep this logging active after the development phase?